src/components/showcase/showcase-section.tsx

```diff
@@ -0,0 +1,213 @@
+
+  const updateRowTranslate = (
+    e: PointerEvent<HTMLAnchorElement> | FocusEvent<HTMLAnchorElement>,
+  ) => {
+    if (!itemsRef.current || isXlDown) return
+
+    const xPosition = e.currentTarget.getBoundingClientRect().left
+    const elementWidth = e.currentTarget.clientWidth
+
+    if (xPosition < 0) {
+      x.set(0)
+    } else if (elementWidth + xPosition > itemsRef.current.clientWidth) {
+      const translateX =
+        -itemsRef.current.scrollWidth + itemsRef.current.clientWidth
+      x.set(translateX)
+    }
+  }
+
+  const ShowcaseItem = ({
+    src,
+    href,
+    alt,
+  }: {
+    src: StaticImageData
+    href: string
+    alt: string
+  }) => {
+    return (
+      <Link
+        isExternal
+        href={href}
+        role='group'
+        onPointerEnter={updateRowTranslate}
+        onFocus={updateRowTranslate}
+        w={{
+          base: 'full',
+          xl: '27.5vw',
+        }}
+        p={2}
+        rounded='md'
+        overflow='hidden'
+        transition='0.25s transform ease-out, 0.25s box-shadow ease-out'
+        _hover={{ transform: 'translateY(-10px)' }}
+        _focus={{
+          transform: 'translateY(-10px)',
+          shadow: 'outline',
+        }}
+      >
+        <VStack position='relative' alignItems='flex-start'>
+          <AspectRatio ratio={16 / 9} w='full'>
+            <ChakraNextImage
+              alt={alt}
+              src={src}
+              layout='fill'
+              objectFit='cover'
+              placeholder='blur'
+              transition='0.25s box-shadow ease-out'
+              _groupHover={{ shadow: 'lg' }}
+              _groupFocus={{ shadow: 'lg' }}
+              rounded='sm'
+            />
+          </AspectRatio>
+          <HStack spacing={6}>
+            <Heading
+              fontSize='md'
+              _groupHover={{ color: 'teal.500' }}
+              _groupFocus={{ color: 'teal.500' }}
+              transition='0.25s color ease-out'
+            >
+              {alt}
+            </Heading>
+          </HStack>
+        </VStack>
+      </Link>
+    )
+  }
+
+  return (
+    <Box as='section' overflow='hidden' mb={12}>
+      <Container py='7.5rem' maxW='1280px'>
+        <Box spacing={5} maxW='760px' mx='auto' textAlign='center'>
+          <chakra.h2 textStyle='heading' mb={4}>
+            {t('homepage.built-with-chakra-section.title')}
+          </chakra.h2>
+          <chakra.p opacity={0.7} fontSize='lg' mb={8}>
+            {t('homepage.built-with-chakra-section.description')}
+          </chakra.p>
+          <NextLink href='/showcase' passHref>
+            <Button
+              h='4rem'
+              px='40px'
+              fontSize='1.2rem'
+              as='a'
+              size='lg'
+              colorScheme='teal'
+              rightIcon={<FaArrowRight fontSize='0.8em' />}
+            >
+              {t('homepage.built-with-chakra-section.see-showcase')}
+            </Button>
+          </NextLink>
```

Comment:
If you use Button with as="a" you could also just do Button as={NextLink} or am I missing something?

Reply:
All of these are external links, so we don't need to use client-side navigation. That's why I used a simple anchor tag instead of `NextLink`.

Reply:
But here it points to our "/showcase" page ?

Reply:
I mean that link in particular. The actual showcase elements I agree

Reply:
The issue with that is that the `Button` component renders multiple children (label, arrow icon), but `NextLink` only supports a single child:
`Multiple children were passed to <Link> with `href` of `/showcase` but only one child is supported`

Reply:
Alright then it's fine for me :)